Add a blog post about TPAC Sevilla (#88)
openwebdocs/owd-website: pages/content/posts/tpac-sevilla.md

---
author: "Florian Scholz"
title: "OWD at TPAC Sevilla"
date: "2023-09-21"
tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "Open Web Docs traveled to Seville, Spain to attend the W3C's annual TPAC conference."
---

Last week I traveled to Seville, Spain to attend the [W3C TPAC 2023 conference](https://www.w3.org/2023/09/TPAC/).

> Every year, the W3C <dfn id="tpac"><abbr title="Technical Plenary and Advisory Committee">TPAC</abbr></dfn> conference invites W3C members and boards, W3C Working and Interest Groups, as well as Invited Experts and others to meet and discuss work on web standards and web technologies. This time it was a hybrid event - participation was possible in-person and remotely.
> 
> It was my first in-person TPAC event and enjoyed meeting a lot of folks that I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.
> I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs) and I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some even watch our [GitHub project repository](https://github.com/openwebdocs/project) to hear all about our new projects immediately (cheers, Thomas Steiner!).
> 
> If you attend an in-person event these days, it really is about the people. So, let me give a shout out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).
> Since our founding, we only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)) and so it was lovely to reconnect and talk about web platform documentation once again.

Reviewer: ```suggestion
Every year, the W3C <dfn id="tpac"><abbr title="Technical Plenary and Advisory Committee">TPAC</abbr></dfn> conference invites W3C members and boards, W3C Working and Interest Groups, as well as Invited Experts and others to meet and discuss work on web standards and web technologies. This time, it was a hybrid event - participation was possible both in-person and remotely.

It was my first in-person TPAC event. I enjoyed meeting a lot of folks that had I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.

I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs). I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some TPAC attendees I spoke with even watch our [GitHub project repository](https://github.com/openwebdocs/project) to immediately hear all about our new projects (cheers, Thomas Steiner!).

Attending an in-person event these days is really about the people. So, let me give a shout-out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).

Since our founding, we have only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)). It was so lovely to reconnect and talk about web platform documentation once again.
```

some grammar suggestions. nothing major. Did you want to link the names to linkedIn or fedi accounts?

## Technical writing breakout session

> To me, the most interesting day was Wednesday. Wednesday was the day for breakout sessions in an unconference style. In collaboration with Rachel and Patrick, I chaired a breakout session: "Getting your feature adopted: learn to work with technical writers" in which we invited the participants to discuss about how to improve technical writing and documenting web technologies. We touched on topics like finding relevant Subject Matter Experts (SMEs) and presented how typically documentation gets written and maintained. [Shawn Lawton Henry](https://www.w3.org/People/Shawn/) (W3C WAI) also let us know there is currently help needed for the [Understanding WCAG](https://www.w3.org/WAI/WCAG22/Understanding/) docs and with Sam MacBeth (DuckDuckGo) we briefly talked about challenges with Webviews that ought to be documented. Find the [full minutes of our session](https://docs.google.com/document/d/1pNHLKwPs0nZ5KCY_cV05unjnL1iHKlb7CW2CQwbGGrs/edit).

Reviewer: ```suggestion
For me, the most interesting day was Wednesday. Wednesday was an unconference-style day of breakout sessions. In collaboration with Rachel and Patrick, I chaired a breakout session, "Getting your feature adopted: learn to work with technical writers." In this session, we invited the participants to discuss how technical writing and documenting web technologies can be improved. We touched on topics like finding relevant Subject Matter Experts (SMEs) and presented how documentation typically gets written and maintained. [Shawn Lawton Henry](https://www.w3.org/People/Shawn/) (W3C WAI) also informed us that help is currently needed for the [Understanding WCAG](https://www.w3.org/WAI/WCAG22/Understanding/) docs. We also discussed the challenges with Webviews that ought to be documented with Sam MacBeth from DuckDuckGo. Read the [full minutes of our session](https://docs.google.com/document/d/1pNHLKwPs0nZ5KCY_cV05unjnL1iHKlb7CW2CQwbGGrs/edit).
```

> One thing to say about chairing and attending TPAC sessions is that I really enjoyed how scribing was taken seriously and how it is possible to be added to the speaker queue by typing "q+" in the IRC channels. You know, as an introvert it can be quite stressful to figure out the right moment to speak up in a room full of (very damn smart) people. Thanks to this, I q+'ed myself, was acknowledged by the chair and could bring up my question or comment. If you are interested in more mechanics of this, I found a blog post on the bocoup blog that has more information: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).

Reviewer: ```suggestion
There were several things I truly enjoyed about chairing and attending TPAC sessions. I really appreciate how scribing was taken seriously. I also appreciated how to be added to the speaker queue, all one has to do is type "q+" in the IRC channels. As an introvert, it can be quite stressful to figure out the right moment to speak up in a room full of (incredibly smart) people. Thanks to this inclusive feature, I q+'ed myself, was acknowledged by the chair, and was able to ask my question or share my perspective. If you are interested in more mechanics of this, I found a blog post on the Bocoup blog: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).
```

## More sessions

> I recently joined the [WebDX W3C community group](https://www.w3.org/community/webdx/) and attended breakout sessions that were related to WebDX:

Reviewer: ```suggestion
I recently joined the [WebDX W3C community group](https://www.w3.org/community/webdx/) and attended breakout sessions related to WebDX:
```

> - Interop project, chaired by James Graham (Mozilla). [Minutes](https://www.w3.org/2023/09/13-interop-minutes.html).
> - Beyond Interop, chaired by Kadir Topal (Google). [Minutes](https://www.w3.org/2023/09/13-beyond-interop-minutes.html).
> - WebDX research and spec development, chaired by François Daoust (W3C). [Minutes](https://www.w3.org/2023/09/13-webdx-minutes.html).
> - Secure the Web Forward, chaired by Daniel Appelquist (Synk). (need to find the minutes link)

Reviewer: ```suggestion
- [Interop project](https://www.w3.org/2023/09/13-interop-minutes.html), chaired by James Graham (Mozilla).
- [Beyond Interop](https://www.w3.org/2023/09/13-beyond-interop-minutes.html), chaired by Kadir Topal (Google).
- [WebDX research and spec development](https://www.w3.org/2023/09/13-webdx-minutes.html), chaired by François Daoust (W3C).
- Secure the Web Forward, chaired by Daniel Appelquist (Synk). (need to find the minutes link)
```
linking the descriptive text.

> Open Web Docs is going to collaborate a lot more the WebDX group going forward. I strongly believe documentation is a major aspect of Developer Experience and the research and work this community group is doing is very relevant to our mission. I met with Daniel Beck and Kadir Topal (Google) to discuss the next steps for the OWD project ["Grouping web features"](https://github.com/openwebdocs/project/issues/169). Daniel also made a video about it, go [watch it](https://www.w3.org/2023/09/TPAC/demos/web-features.html).

Reviewer: ```suggestion
Open Web Docs will be collaborating with the WebDX group a lot more going forward. Documentation is a major aspect of Developer Experience! As such, the research and work this community group is doing is very relevant to OWD's mission. As part of the goal of improving developer experience through better documentation, I met with Daniel Beck and Kadir Topal (Google) to discuss the next steps for the OWD project ["Grouping web features"](https://github.com/openwebdocs/project/issues/169). Daniel also made a [video about web features](https://www.w3.org/2023/09/TPAC/demos/web-features.html).
```

> The "Secure the Web Forward" session was also good preparation the virtual workshop happening soon on September 26-28. See the [agenda](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html). We are looking forward to discuss web security docs and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized that we should probably figure out what would go into an "MDN guide to secure your website / web app". I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post soon about it soon!

Reviewer: ```suggestion
Daniel Appelquist's "Secure the Web Forward" session was also good preparation the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post soon about it soon!
```

## Wrap up

> And that's it! I really enjoyed the face-to-face conversations, even if it was just briefly near the pool with Bert Bos, Brian Kardell, Ali Spivak, Oliver Dunk, Andreu Botella and others which I'm probably forgetting right now (sorry!). Also, happy birthday to Diego González with whom we had a great dinner at a Tapas restaurant.

Reviewer: ```suggestion
And that's it! I really enjoyed the face-to-face conversations, even if it was just briefly near the pool with Bert Bos, Brian Kardell, Ali Spivak, Oliver Dunk, Andreu Botella, and others whom I'm (rudely, sorry!) forgetting right now. Also, a happy birthday shout-out to Diego González. Next year will have to celebrate with another great dinner at a different but equally fantastic Tapas restaurant.
```

> Hoping to see you all at the next TPAC! Let's try to bring the whole OWD team who have had some severe FOMO this year!

Reviewer: ```suggestion
I hope to see you all at TPAC 2024! Next year, let's bring the whole OWD team. They currently have some serious FOMO for missing out on this year's events!
```

---
author: "Florian Scholz"
title: "OWD at TPAC Sevilla"
date: "2023-09-21"
tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "Open Web Docs traveled to Seville, Spain to attend the W3C's annual TPAC conference."
---

Last week I traveled to Seville, Spain to attend the [W3C TPAC 2023 conference](https://www.w3.org/2023/09/TPAC/).

Every year, the W3C <dfn id="tpac"><abbr title="Technical Plenary and Advisory Committee">TPAC</abbr></dfn> conference invites W3C members and boards, W3C Working and Interest Groups, as well as Invited Experts and others to meet and discuss work on web standards and web technologies. This time, it was a hybrid event - participation was possible both in-person and remotely.

> It was my first in-person TPAC event. I enjoyed meeting a lot of folks that had I only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.

Author: ```suggestion
It was my first in-person TPAC event. I enjoyed meeting a lot of folks that I had only interacted with online so far. It was a great feeling when people recognized me or Open Web Docs and had nice things to say.
```

I received kudos for Open Web Docs' setup as a [non-profit Open Collective](https://opencollective.com/open-web-docs). I was also told that folks enjoy reading about the projects we do on MDN Web Docs. Some TPAC attendees I spoke with even watch our [GitHub project repository](https://github.com/openwebdocs/project) to immediately hear all about our new projects (cheers, Thomas Steiner!).

> Attending an in-person event these days is really about the people. So, let me give a shout-out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola, Brian Smith (Mozilla), and Michael[tm] Smith (W3C).

Reviewer 2: ```suggestion
Attending an in-person event these days is really about the people. So, let me give a shout-out to "my" people at TPAC: I connected with a few OWD governing board members, including Rachel Andrew (Google), Daniel Appelquist (Synk), Patrick Brosset (Microsoft), Dominique Hazael-Massieux (W3C), and Robert Nyman (Google). I also met members of OWD's steering committee, including Ruth John (Mozilla), Brian Kardell (Igalia), Lola Odelola (Bocoup), Brian Smith (Mozilla), and Michael[tm] Smith (W3C).
```

Just so everyone is mentioned by name and company.

Since our founding, we have only had a single OWD meetup ([in London, earlier this year](https://openwebdocs.org/content/posts/london-meetup/)). It was so lovely to reconnect and talk about web platform documentation once again.

## Technical writing breakout session

> For me, the most interesting day was Wednesday. Wednesday was an unconference-style day of breakout sessions. In collaboration with Rachel and Patrick, I chaired a breakout session, "Getting your feature adopted: learn to work with technical writers." In this session, we invited the participants to discuss how technical writing and documenting web technologies can be improved. We touched on topics like finding relevant Subject Matter Experts (SMEs) and presented how documentation typically gets written and maintained. [Shawn Lawton Henry](https://www.w3.org/People/Shawn/) (W3C WAI) also informed us that help is currently needed for the [Understanding WCAG](https://www.w3.org/WAI/WCAG22/Understanding/) docs. We also discussed the challenges with Webviews that ought to be documented with Sam MacBeth from DuckDuckGo. Read the [full minutes of our session](https://docs.google.com/document/d/1pNHLKwPs0nZ5KCY_cV05unjnL1iHKlb7CW2CQwbGGrs/edit).

Reviewer: ```suggestion
For me, the most interesting day was Wednesday. Wednesday was an unconference-style day of breakout sessions. In collaboration with Rachel and Patrick, I chaired a breakout session, "Getting your feature adopted: learn to work with technical writers." In this session, we invited the participants to discuss how technical writing and documenting web technologies can be improved. We touched on topics like finding relevant Subject Matter Experts (SMEs) and presented how documentation typically gets written and maintained. [Shawn Lawton Henry](https://www.w3.org/People/Shawn/) (W3C WAI), a former tech writer, joined us. She informed us that help is currently needed for the [Understanding WCAG](https://www.w3.org/WAI/WCAG22/Understanding/) docs. We also discussed the challenges with Webviews that ought to be documented with Sam MacBeth from DuckDuckGo. Read the [full minutes of our session](https://docs.google.com/document/d/1pNHLKwPs0nZ5KCY_cV05unjnL1iHKlb7CW2CQwbGGrs/edit).
```

There were several things I truly enjoyed about chairing and attending TPAC sessions. I really appreciate how scribing was taken seriously. I also appreciated how to be added to the speaker queue, all one has to do is type "q+" in the IRC channels. As an introvert, it can be quite stressful to figure out the right moment to speak up in a room full of (incredibly smart) people. Thanks to this inclusive feature, I q+'ed myself, was acknowledged by the chair, and was able to ask my question or share my perspective. If you are interested in more mechanics of this, I found a blog post on the Bocoup blog: [How to scribe at TPAC](https://bocoup.com/blog/how-to-scribe-at-tpac).

## More sessions

I recently joined the [WebDX W3C community group](https://www.w3.org/community/webdx/) and attended breakout sessions related to WebDX:

- [Interop project](https://www.w3.org/2023/09/13-interop-minutes.html), chaired by James Graham (Mozilla).
- [Beyond Interop](https://www.w3.org/2023/09/13-beyond-interop-minutes.html), chaired by Kadir Topal (Google).
- [WebDX research and spec development](https://www.w3.org/2023/09/13-webdx-minutes.html), chaired by François Daoust (W3C).

> - Secure the Web Forward, chaired by Daniel Appelquist (Synk). (need to find the minutes link)

Reviewer 3: ```suggestion
- [Secure the Web Forward](https://www.w3.org/2023/09/13-secure-the-web-minutes.html), chaired by Daniel Appelquist (Synk).
```

Open Web Docs will be collaborating with the WebDX group a lot more going forward. Documentation is a major aspect of Developer Experience! As such, the research and work this community group is doing is very relevant to OWD's mission. As part of the goal of improving developer experience through better documentation, I met with Daniel Beck and Kadir Topal (Google) to discuss the next steps for the OWD project ["Grouping web features"](https://github.com/openwebdocs/project/issues/169). Daniel also made a [video about web features](https://www.w3.org/2023/09/TPAC/demos/web-features.html).

> Daniel Appelquist's "Secure the Web Forward" session was also good preparation the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post soon about it soon!

Author: ```suggestion
Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post soon it soon!
```

> Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are very eager to start discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". I'm curious to see how the format will play out, but I'm sure this workshop will be invaluable in reaching this project's goals. Expect a blog post about it soon!

Reviewer: ```suggestion
The "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are eager to discuss web security documentation and lessons with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for an "MDN guide to secure your website/web app". I am curious to see how the format will play out, but know this workshop will be invaluable in reaching this project's goals. Expect a blog post about it soon!
```

> Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post about it soon!

Reviewer 2: nit: the word "forward" is repeated 3 times at the start of this paragraph. Maybe find a way to remove the third instance. E.g "We're very eager to start discussing web security ..."

> Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are looking forward to discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". This workshop will be invaluable in reaching this project's goals. I'm very much looking forward to this workshop and I'm curious to see how the format will play out. Expect a blog post about it soon!

Author: ```suggestion
Daniel Appelquist's "Secure the Web Forward" session was also good preparation for the [Secure the Web Forward virtual workshop](https://www.w3.org/2023/03/secure-the-web-forward/agenda.html) happening September 26-28. We are very eager to start discussing web security documentation and security education with security experts. While talking to Arnaud Le Hors (IBM), I realized we should likely start our security docs project by outlining the contents for a "MDN guide to secure your website / web app". I'm curious to see how the format will play out, but I'm sure this workshop will be invaluable in reaching this project's goals. Expect a blog post about it soon!
```

## Wrap up

> And that's it! I really enjoyed the face-to-face conversations, even if it was just briefly near the pool with Bert Bos, Brian Kardell, Ali Spivak, Oliver Dunk, Andreu Botella, and others whom I'm (rudely, sorry!) forgetting right now. Also, a happy birthday shout-out to Diego González. Next year will have to celebrate with another great dinner at a different but equally fantastic Tapas restaurant.

Reviewer 2: ```suggestion
And that's it! I really enjoyed the face-to-face conversations, even if it was just briefly near the pool with Bert Bos, Brian Kardell, Ali Spivak, Oliver Dunk, Andreu Botella, and others whom I'm (rudely, sorry!) forgetting right now. Also, a happy birthday shout-out to Diego González. Next year we'll have to celebrate with another great dinner at a different but equally fantastic Tapas restaurant.
```

> I hope to see you all at TPAC 2024! Next year, let's bring the whole OWD team. They currently have some serious FOMO for missing out on this year's events!

Author: ```suggestion
I hope to see you all at TPAC 2024! Next year, let's bring the whole OWD team. They currently have some serious FOMO for missing out on this year's event!
```
